tests/system/test_dbapi.py

```diff
@@ -1339,31 +1375,23 @@ def test_json_array(self, dbapi_database):
                 op = dbapi_database.update_ddl(["DROP TABLE JsonDetails"])
                 op.result()
 
-    @pytest.mark.noautofixt
-    def test_DDL_commit(self, shared_instance, dbapi_database):
+    def test_DDL_commit(self, dbapi_database):
         """Check that DDLs in commit mode are executed on calling `commit()`."""
```

Comment:
I think that this also is only true if `buffer_ddl=True`. In other cases, this would cause an error (?)

Reply:
Thats correct. Added the comment